travis/unit_tests_default.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 # Test default installation
-pytest fitbenchmarking/cli --cov=fitbenchmarking/cli --cov-report term-missing
+if [ -z "$1" ]
```

Comment:
From our discussion this should be reverted back to what it was before as optional arguments are no longer needed